save_text_report.py

```
@@ -0,0 +1,136 @@
+    with open(analysis_file, 'r', encoding='utf-8') as f:
+        data = json.load(f)
+    
+    # Form report
+    report = []
+    report.append("=" * 80)
+    report.append("DANCE ANALYSIS RESULTS")
+    report.append("=" * 80)
+    report.append("")
+    report.append(f"Figure: {data['classification']['figure']}")
+    report.append(f"Confidence: {data['classification']['confidence']:.1%}")
+    report.append(f"")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)